contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +394,4 @@
 </form>
 <?php
 formFooter();
-?>
+?> 
```

Comment:
I think that is more geared for pure php libraries that are included at the top of the script where spaces outside the php could break things.